plugins/catalog-backend/src/next/NextRouter.ts

```diff
@@ -133,8 +133,8 @@ export async function createNextRouter(
         res.status(201).json(output);
       })
       .get('/locations', async (_req, res) => {
-        const output = await locationService.listLocations();
-        res.status(200).json(output);
+        const locations = await locationService.listLocations();
+        res.status(200).json(locations.map(l => ({ data: l })));
```

Comment:
```suggestion
        // TODO(freben): This currently leaves out the currentStatus field, that was
        // in the old version of the router. E.g.
        // "currentStatus":{"message":null,"status":"success","timestamp":"2021-05-10 08:47:06"}
        // We may want to reintroduce it by joining the locations table with the refresh state, or
        // alternatively by at least generating a ref to the generated entity, so that the user can
        // look up errors via the /entities API instead
        res.status(200).json(locations.map(l => ({ data: l })));
```

Reply:
i don't see this text in here